src/controller/agent.controller.service.ts

```diff
@@ -47,7 +47,7 @@ export class AgentControllerService {
         if (process.env.MULTI_AGENT === 'true') {
             agentUrl = process.env.MULTITENANT_URL;
         } else {
-            const adminPort = ((profile && profile.adminApiPort) ? profile.adminApiPort : process.env.AGENT_ADMIN_PORT);
+            const adminPort: string = ((profile && profile.adminApiPort) ? profile.adminApiPort : process.env.AGENT_ADMIN_PORT);
```

Comment:
I need to be convinced its a good thing to remove type delcarations

Reply:
We're adding a type declaration here, though.

Reply:
agreed.  I just want to have the conversation in general.  Ill mark this as resolved, specifically.